content_sync/management/commands/trigger_pipelines.py

```diff
@@ -62,7 +62,14 @@ def handle(self, *args, **options):
         for website in website_qset.iterator():
             pipeline = get_sync_pipeline(website)
             try:
-                pipeline.trigger_pipeline_build(version)
+                build_id = pipeline.trigger_pipeline_build(version)
+                Website.objects.filter(pk=website.pk).update(
+                    **{
+                        "latest_build_id_draft"
+                        if version == "draft"
```

Comment:
Could use `BaseSyncPipeline.VERSION_DRAFT` and `BaseSyncPipeline.VERSION_LIVE` instead of "draft" and "live", here and elsewhere.